src/kw_env.sh

```diff
@@ -54,6 +58,34 @@ function env_main()
   fi
 }
 
+# If the env option is requested, it means the O= option from the kernel
+# makefile is used, and this option requires a clean kernel tree to work as
+# expected. This function checks if the current kernel tree is clean.
+#
+# Return:
+# Return 22 in case of error
+function validate_env_before_use()
```

Comment:
I don't know much about the `kw env` feature, but what do you think about the name `validate_env_before_switch`?

Reply:
I'll use that.